src/connectedk8s/azext_connectedk8s/_help.py

```diff
@@ -131,3 +131,11 @@
   - name: Perform diagnostic checks on an Arc enabled Kubernetes cluster.
     text: az connectedk8s troubleshoot -n clusterName -g resourceGroupName
 """
+
+helps['connectedk8s show-issuer-url'] = """
```

Comment:
should this go into the "connectedk8s show" command itself with --show-issuer-url as a flag to the command instead of a new command?

cc: @omkark95

Reply:
Since this is a temporary command that will only be used for private preview in a whl file and will need to be removed later I thought it was best to keep this separate from all other code.